tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -204,3 +204,10 @@ tests:
   drivers.spi.max32_dma.loopback:
     extra_args: OVERLAY_CONFIG="overlay-max32-spi-dma.conf"
     filter: CONFIG_SOC_FAMILY_MAX32
+  drivers.spi.max32_spi_rtio.loopback:
+    extra_args:
+      - OVERLAY_CONFIG="overlay-max32-spi-rtio.conf"
+      - DTC_OVERLAY_FILE="boards/apard32690_max32690_m4.overlay"
```

Comment:
Shouldn't need to specify this explicitly because the build system should pick it up automatically